public/app/features/logs/components/LogRow.tsx

```diff
@@ -191,10 +198,7 @@ class UnThemedLogRow extends PureComponent<Props, State> {
       [styles.highlightBackground]: permalinked && !this.state.showDetails,
     });
 
-    const processedRow =
-      row.hasUnescapedContent && forceEscape
-        ? { ...row, entry: escapeUnescapedString(row.entry), raw: escapeUnescapedString(row.raw) }
-        : row;
+    const processedRow = this.processRow(row, forceEscape);
```

Comment:
I know `processedRow` was the name before, but what do you think about being a bit more explicit, in both the variable and method name? E.g. `escapedRow`/`escapeRow`?